### ISSUE TITLE
ConvertFrom-Clixml should support conversions from files holding multiple Clixml objects

### ISSUE THREAD

**Issue by techthoughts2** [austoonz/Convert#4]
### Expected Behavior

When running ConvertFrom-Clixml with a file that contains multiple Clixml objects - the conversion should be able to convert all objects via the pipeline.

```powershell
$galleryXMLData = Get-Content -Path "C:\xml\all.xml" -Raw
$converted = $galleryXMLData | ConvertFrom-Clixml
```

### Current Behavior

Currently - regardless if ```-Raw``` is used or not ```ConvertFrom-Clixml``` is only able to convert a file if it contains a singular Clixml object.

### Possible Solution

Here is how I am currently working around this:

```powershell
$galleryData = @()
$galleryXMLData = Get-Content -Path "C:\xml\all.xml" -Raw
$splitup = $galleryXMLData -split '(?<!^)(?=<Objs Version="1.1.0.1" xmlns="http:\/\/schemas.microsoft.com\/powershell\/2004\/04">)'
foreach ($item in $splitup) {
    $galleryData += $item | ConvertFrom-Clixml
}
```

### Steps to Reproduce

1. Download the [Sample XML you can use for testing](http://techthoughts.info/shareout/all.zip)
2. Import the file data using ```Get-Content```
3. Attempt to convert the contents of the data back into PS Object format

### Context (Environment)

I have a Clixml file that contains many converted objects. I am attempting to get the contents of the file and convert them back to PowerShell object format. This is working when the file contains one Clixml entry, but not when it contains multiple.

I do currently have a work-around for this which I have shown above.

* Operating System and version as reported by `$PSVersionTable.OS`: ```Microsoft Windows 10.0.18362```
* PowerShell versions as reported by `$PSVersionTable.PSEdition`: ```Core```

### Detailed Description

Based on the behavior it seems like Get-Content doesn't have a delimter available for Clixml data to properly parse it. Because of the lack of delimter the ```ConvertFrom-Clixml``` doesn't have a way to handle multiple objects.

This could be handled by introducing a delimeter of some kind when ```ConvertTo-Clixml``` is used. Alternatively, logic could be added to permit ```ConvertFrom-Clixml``` to handle multi-object conversions.

**Comment by austoonz** [comment]
Should be resolved in v0.4.1 on the Gallery. Thanks for the regex... leveraged part of that to help! 👍

**Comment by techthoughts2** [comment]
Fix confirmed working. This can be closed out. Thanks for the quick response!
